e2e/test/MessageSendE2EMultiplexingTests.cs

```diff
@@ -13,18 +13,16 @@ namespace Microsoft.Azure.Devices.E2ETests
 {
     [TestClass]
     [TestCategory("IoTHub-E2E")]
-    [TestCategory("ConnectionPooling_AMQP_E2E")]
```

Comment:
1. Why do we need another category.
2. Why is it different from the class name?

Reply:
This is a revert; these categories are now removed.